### PR TITLE
Some Windows/MSYS2-specific fixes

### PR DESCRIPTION
**Pull request by hgdagon**
These changes _shouldn't_ affect other platforms, but I guess the checks will show.

### CODE REVIEW

File: revision.h.in [hunk 1]
```diff
@@ -18,11 +18,8 @@
 #else
 #define VER_FILEVERSION_STR         STR(VER_MAJOR) "." STR(VER_MINOR) "." STR(VER_PATCH) "." STR(VER_BUILD) " (Commit @GIT_COMMIT_HASH@)"
 #endif
+#define VER_COPYRIGHT_STR           "� 2006-@THIS_YEAR@"
```
Comment (RichardTea) [inline revision.h.in:21]: Avoid non-ascii characters in source code and especially string literals.
Not all editors support them - including github - and of course the native unicode string encoding varies between platforms.
Just use (C).

**Review by RichardTea — CHANGES_REQUESTED** [review]

File: revision.h.in (hunk 1 above)
Comment (hgdagon) [reply revision.h.in:21]: Found ANSI escape sequence. Fixed.

File: CMakeLists.txt [hunk 2]
```diff
@@ -126,7 +126,7 @@ OPTION ( ASSIMP_BUILD_DOCS
 )
 OPTION( ASSIMP_INJECT_DEBUG_POSTFIX
   "Inject debug postfix in .a/.so/.dll lib names"
-  ON
+  OFF
```
[the remark this answers is not in the document]
Comment (hgdagon) [reply CMakeLists.txt:129]: Whoops... Leftover from tests.

File: CMakeLists.txt [hunk 3]
```diff
@@ -720,12 +708,24 @@ IF ( ASSIMP_BUILD_TESTS )
...
 ENDIF ()
 
-# Generate a pkg-config .pc for the Assimp library.
+# Generate a pkg-config .pc, revision.h, and config.h for the Assimp library.
 CONFIGURE_FILE( "${PROJECT_SOURCE_DIR}/assimp.pc.in" "${PROJECT_BINARY_DIR}/assimp.pc" @ONLY )
 IF ( ASSIMP_INSTALL )
   INSTALL( FILES "${PROJECT_BINARY_DIR}/assimp.pc" DESTINATION ${ASSIMP_LIB_INSTALL_DIR}/pkgconfig/ COMPONENT ${LIBASSIMP-DEV_COMPONENT})
 ENDIF()
 
+string(TIMESTAMP THIS_YEAR "%Y")
```
[the remark this answers is not in the document]
Comment (hgdagon) [reply CMakeLists.txt:717]: Yeah. I  wasn't too confident about this either. Replaced with a direct string in `revision.h`

File: revision.h.in [hunk 4]
```diff
@@ -18,11 +18,8 @@
...
 
-#ifdef  NDEBUG
-#define VER_ORIGINAL_FILENAME_STR   "assimp@LIBRARY_SUFFIX@.dll"
-#else
-#define VER_ORIGINAL_FILENAME_STR   "assimp@LIBRARY_SUFFIX@@CMAKE_DEBUG_POSTFIX@.dll"
-#endif //  NDEBUG
+#define VER_ORIGINAL_FILENAME_STR   "@CMAKE_SHARED_LIBRARY_PREFIX@assimp@LIBRARY_SUFFIX@@CMAKE_DEBUG_POSTFIX@.dll"
```
[the remark this answers is not in the document]
Comment (hgdagon) [reply revision.h.in:23]: Darn! I had completely forgotten about multi-config generators.

**Comment by hgdagon** [comment]
Just a little note, that a few of the changes are in preparation to another PR related to .rc files.

**Pull request closed without merging**